scenes/game_logic/fill_game_logic.gd

```diff
@@ -14,7 +14,9 @@ func start() -> void:
 	var player: Player = get_tree().get_first_node_in_group("player")
 	if player:
 		player.mode = Player.Mode.FIGHTING
-	get_tree().call_group("throwing_enemy", "start")
+	for throwing_enemy: ThrowingEnemy in get_tree().get_nodes_in_group("throwing_enemy"):
+		if not throwing_enemy.autostart:
+			throwing_enemy.start()
```

Comment:
This works but you might have considered making `start()` idempotent, i.e. a no-op if it has already been called.